acme/messages2_test.py

```diff
@@ -32,15 +33,15 @@ def test_typ_prefix(self):
 
     def test_typ_decoder_missing_prefix(self):
         from acme.messages2 import Error
-        self.assertRaises(jose.DeserializationError, Error.from_json,
-                          {'detail': 'foo', 'type': 'malformed'})
-        self.assertRaises(jose.DeserializationError, Error.from_json,
-                          {'detail': 'foo', 'type': 'not valid bare type'})
+        self.jobj['type'] = 'malfomed'
+        self.assertRaises(jose.DeserializationError, Error.from_json, self.jobj)
+        self.jobj['type'] = 'not balid bare type'
```

Comment:
Also "malformed" instead of "malfomed" (although it might well raise the same exception and pass the test!).